test/test.dhall

```diff
@@ -0,0 +1,10 @@
+let conf = ../spago.dhall
```

Comment:
The separate `test.dhall` is so we can depend on `spec` just for the test suite.

However, this does mean that `spago -x test/test.dhall test` must be used instead of just `spago test`.

Is this the right way to do this, or is there a better/more idiomatic way of handling this?

Reply:
This change is also why CI is currently failing.

Reply:
Everything should just go in the main Spago.dhall file, including test dependencies. They aren’t used when the library is published (the spago.dhall in package sets is used), so it’s ok.

This will all be updated when the registry gets published, so just adding the dependency and making sure the tests are in sources is the way to go for now 👍